src/main/java/com/avairebot/commands/fun/UndertaleTextBoxCommand.java

```diff
@@ -0,0 +1,99 @@
+package com.avairebot.commands.fun;
+
+import com.avairebot.AvaIre;
+import com.avairebot.commands.CommandMessage;
+import com.avairebot.contracts.commands.Command;
+import com.avairebot.language.I18n;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.MessageBuilder;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class UndertaleTextBoxCommand extends Command {
+
+    private final String templateUrl = "https://www.demirramon.com/gen/undertale_box.png?character={0}&message={1}&ext=.png";
+
+    public UndertaleTextBoxCommand(AvaIre avaire) {
+        super(avaire);
+    }
+
+    @Override
+    public String getName() {
+        return "Undertale TextBox Command";
+    }
+
+    @Override
+    public String getDescription() {
+        return "TODO";
```

Comment:
Should have a proper description.

Reply:
I gave it the following description. 

Create your own Undertale text boxes with any character, expression and text!  Generator owned by  Demirramon. Undertale owned by Toby Fox. All rights reserved.